### PR TITLE
add yamlfmt to all-repos

### PR DESCRIPTION
**Pull request by braydonk**
my new repository:

- [x] is added to the bottom *or* with existing repos from the same account
- [x] contains a license
- [x] is not `language: system` or `language: docker`
- [x] does not contain "pre-commit" in the name

### CODE REVIEW

File: all-repos.yaml [hunk 1]
```diff
@@ -227,3 +227,4 @@
 - https://github.com/charliermarsh/ruff-pre-commit
 - https://github.com/mrtazz/checkmake
 - https://github.com/jshwi/docsig
+- https://github.com/google/yamlfmt
```
Comment (asottile) [inline all-repos.yaml:230]: `yamlfmt .` is going to double-run every file (pre-commit works by passing the files as positional arguments to hooks).  there are two choices: (1) remove the `.` (recommended) or (2) set `pass_filenames: false` (I'm not accepting new things to the list which sets this)

Comment (braydonk) [reply all-repos.yaml:230]: Makes sense, I'll remove that and re-open this PR on the next release.

**Comment by braydonk** [comment]
I'm reopening this PR for consideration. I made the recommended adjustment by removing `.` from `entry`.

File: all-repos.yaml (hunk 1 above)
Comment (djgoku) [reply all-repos.yaml:236]: Looks like `v0.8.0` was tagged and removes the `.` [1].

[1] https://github.com/google/yamlfmt/blob/v0.8.0/.pre-commit-hooks.yaml

Comment (asottile) [reply all-repos.yaml:236]: nice!  can you move this up next to the other `google/` hook?

Comment (braydonk) [reply all-repos.yaml]: Ah good idea, done in [afeac4c](https://github.com/pre-commit/pre-commit.com/pull/772/commits/afeac4c645a9276353d037f65e87807f01670c84)!

**Review by asottile — APPROVED** [review]

**Pull request merged**